csharp/wavefrontsender.md

```diff
@@ -65,7 +65,7 @@ IWavefrontSender wavefrontSender = wfProxyClientBuilder.Build();
 When sending data via direct ingestion, you need to create a `WavefrontDirectIngestionClient`, and build it with the Wavefront URL and API token to send data directly to Wavefront.
 
 >**Prerequisite**
-> * Verify that you have the Direct Data Ingestion permission. For details, see [Examine Groups and Permissions](https://docs.wavefront.com/users_account_managing.html#examine-groups-and-permissions).
+> * Verify that you have the Direct Data Ingestion permission. For details, see [Examine Groups and Permissions](https://docs.wavefront.com/users_account_managing.html#examine-groups-roles-and-permissions).
```

Comment:
Title of the section is now "Examine Groups, Roles, and Permissions"

Reply:
Thank you! Updated the title.